demos/m5/enterprise_contract_policy_with_ignores.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: appstudio.redhat.com/v1alpha1
```

Comment:
Is this file intended to be here? I can't imagine we want two `m5-policy`s in the `managed` namespace

Reply:
It is. The one in base produces errors (required in the demo). The one in the m5 directory adds some ignores to pass the validation.